airflow/www/forms.py

```diff
@@ -158,6 +163,59 @@ def populate_obj(self, item):
             item.conf = json.loads(item.conf)
 
 
+class DagRunEditForm(DynamicForm):
```

Comment:
Inherit from DagRun form maybe? conf could be editable too if we wanted it to be.

Reply:
done in https://github.com/apache/airflow/pull/12770/commits/8c969c85aecc3305e253d65f16b111c3b94a48ab

I don't think we should make conf editable as it should show the conf value passed when the DagRun was created like we do when someone clicks on "Add new DagRun"